### ISSUE TITLE
When SLB reopens it is on pause rather than play

### ISSUE THREAD

**Issue by CastonWebDesigns** [archetyped/simple-lightbox#553]
Thank you for your really quick reply on issue #552. Magnific Popup is built in to the Divi theme and I now see a number of users have had the same problem and disabled it which I’ve now done and it is working well with one very small issue which if you can solve or help me with I’ll be grateful. Click on an image and the lightbox opens and is on play. Once you close the lightbox when it reopens it is on pause. Can it re-open on play or is that not possible?

**Comment by archetyped** [comment]
Yes, resuming the slideshow play state is planned for a future update.
